services/intersection-api/api/src/main/java/us/dot/its/jpo/ode/api/accessors/bsm/OdeBsmJsonRepositoryImpl.java

```diff
@@ -15,24 +16,23 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 import us.dot.its.jpo.geojsonconverter.DateJsonMapper;
-import us.dot.its.jpo.ode.api.ConflictMonitorApiProperties;
 import us.dot.its.jpo.ode.model.OdeBsmData;
 
 @Component
 public class OdeBsmJsonRepositoryImpl implements OdeBsmJsonRepository {
 
     private final MongoTemplate mongoTemplate;
-    private final ConflictMonitorApiProperties props;
+
+    @Value("${maximumResponseSize}")
+    int maximumResponseSize;
 
     private final ObjectMapper mapper = DateJsonMapper.getInstance()
             .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
     private final String collectionName = "OdeBsmJson";
 
     @Autowired
-    public OdeBsmJsonRepositoryImpl(MongoTemplate mongoTemplate,
-            ConflictMonitorApiProperties props) {
+    public OdeBsmJsonRepositoryImpl(MongoTemplate mongoTemplate) {
```

Comment:
chore(blocking): use constructor injection
```suggestion
    public OdeBsmJsonRepositoryImpl(MongoTemplate mongoTemplate, @Value("${maximumResponseSize}") int maximumResponseSize) {
    this.maximumResponseSize = maximumResponseSize;
```

Reply:
No longer necessary, as maximumResponseSize is handled by the controllers